README.md

```diff
@@ -46,25 +46,18 @@ To define a plugin-level environment setup script, you can add lines like the fo
 </processingTools>
 ```
 
-This will declare that the file `resources/workflowName/environments/conda.sh` to be used as workflow setup
-script for all jobs. Like all Roddy "tools" such environment scripts need to be executable.
+This will declare that the file `resources/workflowName/environments/conda.sh` to be used as workflow setup script for all jobs. Like all Roddy "tools" such environment scripts need to be executable.
```

Comment:
This will declare the file ... to be used
or
This will declare that the file ... is used